core/src/main/scala/spinal/core/AFix.scala

```diff
@@ -592,7 +592,7 @@ class AFix(val maxRaw: BigInt, val minRaw: BigInt, val exp: Int) extends MultiDa
     val ret = new AFix(this.maxRaw / shiftBig, this.minRaw / shiftBig, this.exp)
 
     if (this.signed)
-      ret.raw := this.raw.dropLow(shift)
+      ret.raw := (this.raw.asSInt >> shift).asBits
```

Comment:
I would say this change isn't required.
Are you sure the old implementation was bugy ?
I mean, ret.raw := this.raw.dropLow(shift)  only remove the LSB bits. No padding is required. (the operator comment is wrong XD about "adding padding bits left")

Reply:
Else it is good for merge :)

Reply:
I'll take another look at this when I'm done with my current task.